doc/source/ray-core/fault_tolerance/ray_components.rst

```diff
@@ -0,0 +1,69 @@
+.. _fault-tolerance-internal-system:
+
+Advanced topic: Ray system failure model
+========================================
+
+Here, we describe the system-level components of Ray and how failures in those
+components can affect tasks, objects, and actors.
+
+.. _fault-tolerance-gcs:
+
+GCS failure
```

Comment:
I think the section order should be worker failure, raylet failure, gcs failure (last).